ocaml/idl/datamodel_host.ml

```diff
@@ -625,12 +625,46 @@ let disable =
         , "Puts the host into a state in which no new VMs can be started. \
            Currently active VMs on the host continue to execute."
         )
+      ; ( Changed
+        , "25.31.0"
+        , "Added host_disabled_until_reboot option to allow persisting the \
+           state after toolstack restarts. Added host_disabled_across_reboot \
+           option to allow disabling the host indefinitely until manually \
+           re-enabled."
+        )
       ]
     ~name:"disable"
     ~doc:
       "Puts the host into a state in which no new VMs can be started. \
        Currently active VMs on the host continue to execute."
-    ~params:[(Ref _host, "host", "The Host to disable")]
+    ~versioned_params:
+      [
+        {
+          param_type= Ref _host
+        ; param_name= "host"
+        ; param_doc= "The Host to disable"
+        ; param_release= rio_release
+        ; param_default= None
+        }
+      ; {
+          param_type= Bool
+        ; param_name= "host_disabled_until_reboot"
```

Comment:
The current `host.disable` is equivalent to `host_disabled_until_reboot=true, host_disabled_across_reboot=false`. Would there be only two cases: across reboot or not?
We'd better not change the semantic of the existing `host.disable` which is actually `host_disabled_until_reboot=true`. Otherwise, the clients would fail on expectations.